src/dataconnect/schemaMigration.ts

```diff
@@ -205,44 +199,51 @@ async function handleIncompatibleSchemaError(args: {
 async function promptForSchemaMigration(
   options: Options,
   databaseName: string,
-  err: IncompatibleSqlSchemaError,
-  allowNonInteractiveMigration: boolean,
-): Promise<"none" | "safe" | "all"> {
+  err: IncompatibleSqlSchemaError | undefined,
+  validateOnly: boolean,
+): Promise<"none" | "all"> {
```

Comment:
Since there's only two options now, should this return `Promise<false | true>`? Also does `maybeExecuteSchemaMigration` better describe what this method does?

Reply:
This method doesn't execute schema migration. It determines whether to execute it later.

Well, keeping it as string merely because we will have a "compatible" mode soon. At least 3 value enum is needed here.